pipeline.mmd

```diff
@@ -0,0 +1,19 @@
+graph TD
+  subgraph Airflow
+    subgraph Collect and convert to CSV
+      ejp-xml["eJP XML"] --> csv-generator["CSV generator"]
+      ejp-csv["eJP CSV"] --> csv-generator
+      observer["Observer"] --> csv-generator
```

Comment:
this is interesting to me, as Observer is a purely derived data source

Reply:
It was originally planned for the initial prototype. But to fully replicate the editorial reports we need the publication dates which seemed simple enough to get out of observer (that's where the current reports get it from). We could consider going to the underlying data source in the future.

Reply:
Observer is a bit of a second class citizen at elife. it doesn't enjoy a CDN and doesn't have an API for access consistent with the one used by the rest of the infrastructure. The RSS feeds on the journal are linked directly I believe. Let me know before you start enumerating Observer results, it's never really experienced any load.

I don't see Observer going away, but chaining the scraping so you're scraping a scrape seems like chinese whispers.